src/patternfly/components/OverflowMenu/overflow-menu.scss

```diff
@@ -1,89 +1,51 @@
-// @debug $overflow-menu; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
+:where(:root),
+:where(.#{$overflow-menu}) {
+  --#{$overflow-menu}--ColumnGap: var(--pf-t--global--spacer--sm);
 
-.#{$overflow-menu} {
-  // Base spacer - shared value
-  --#{$overflow-menu}--spacer--base: var(--#{$pf-global}--spacer--md);
-
-  // Spacer values
-  --#{$overflow-menu}--spacer: var(--#{$pf-global}--spacer--sm);
-  --#{$overflow-menu}__group--spacer: var(--#{$overflow-menu}--spacer--base);
-  --#{$overflow-menu}__item--spacer: var(--#{$overflow-menu}--spacer--base);
-  --#{$overflow-menu}--c-divider--m-vertical--spacer: var(--#{$overflow-menu}--spacer--base);
-
-  // Button group
-  --#{$overflow-menu}__group--m-button-group--spacer: var(--#{$overflow-menu}--spacer--base);
-  --#{$overflow-menu}__group--m-button-group--space-items: var(--#{$pf-global}--spacer--sm);
+  // * Overflow menu icon button group
+  --#{$overflow-menu}__group--m-icon-button-group--ColumnGap: 0;
+}
 
-  // Icon button group
-  --#{$overflow-menu}__group--m-icon-button-group--spacer: var(--#{$overflow-menu}--spacer--base);
-  --#{$overflow-menu}__group--m-icon-button-group--space-items: 0;
+// - Overflow menu - Overflow menu content - Overflow menu group
+.#{$overflow-menu},
+.#{$overflow-menu}__content,
+.#{$overflow-menu}__group {
+  display: flex;
+}
 
-  display: inline-flex;
-  align-items: center;
+// - Overflow menu
+.#{$overflow-menu} {
+  column-gap: var(--#{$overflow-menu}--ColumnGap);
+  width: fit-content;
 }
 
-// Content
+// - Overflow menu content
 .#{$overflow-menu}__content {
-  display: flex;
-  align-items: center;
+  align-items: start;
 }
 
-// Group
+// - Overflow menu group
 .#{$overflow-menu}__group {
-  --#{$overflow-menu}--spacer: var(--#{$overflow-menu}__group--spacer);
-
-  display: flex;
-  align-items: center;
+  column-gap: var(--#{$overflow-menu}__group--ColumnGap, var(--#{$overflow-menu}--ColumnGap));
```

Comment:
Might be similar to Michael's comment(s) above? I think one worry about doing this without defining the var is that it doesn't get documented on Org, so you have to inspect the page to see that `--#{$overflow-menu}__group--ColumnGap` isn't being applied to the column gap.

Can be a follow up if we would want to change it, but lmk if there's another reason for doing this vs defining --#{$overflow-menu}__group--ColumnGap and other vars at the top

Reply:
Doesn't look like the workspace is currently generating css variables correctly as is, so that requires a followup. 

@thatblindgeye Great point, in this case, I agree with you. In other cases, where something like `menu` shares `row-gap` values, I much prefer applying this fallback method.